service/static/index.html

```diff
@@ -176,7 +175,7 @@ <h2 class="">Results</h2>
     </form>
 
     <!-- Search Results -->
-      <div class="table-responsive col-md-12" id="search_results">
```

Comment:
Just fyi, this wasn't doing anything since everything spans 12 columns by default. I can explain further if anyone would like.